### PR TITLE
Add learnings from tax-tribunals DB setup

### PR DESCRIPTION
**Pull request by WillTaylor22**

### CODE REVIEW

File: source/onboarding/person/index.html.md.erb [hunk 1]
```diff
@@ -9,6 +9,8 @@ weight: 2
 
 ## Azure AD groups
 
+Azure AD is used to give you and your team-mates access to specific subscriptions and resources in Azure. When your github user has access to a subscription, you will be able to view, create and modify Azure resources for your team.
```
Comment (timja) [inline source/onboarding/person/index.html.md.erb:12]: use sentence per line to make diff's easier to read, and use the product's display name, GitHub, not github
```suggestion
Azure AD is used to give you and your team-mates access to specific subscriptions and resources in Azure.
When your GitHub user has access to a subscription, you will be able to view, create and modify Azure resources for your team.
```

File: source/onboarding/team/azuread.html.md.erb [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+Azure AD is used to give you and your team-mates access to specific subscriptions in Azure. When your github user has access to a subscription, you will be able to view, create and modify Azure resources for your team.
 
-Create a help request in the [#platops-help](https://hmcts-reform.slack.com/messages/platops-help) channel on Slack.
 
-Description:
+## Access to a an existing team.
 
-> Hi
-> I have just started on the "team name" team,
+Follow the instructions [here](http://127.0.0.1:4567/onboarding/person/#azure-ad-groups)
```
Comment (timja) [inline source/onboarding/team/azuread.html.md.erb:15]: avoid using here, it's not very accessible, prefer the page title or a better description, also localhost isn't going to work off your machine

```suggestion
Follow the instructions on the [person onboarding guide](/onboarding/person/#azure-ad-groups).
```

File: source/onboarding/team/azuread.html.md.erb [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
+
+## Creating a new team
 
-> Can you please create a team AAD group: "DTS Team name"
+If you are unsure which groups you need to be a part of, ask your team or create a help request in the [#platops-help](https://hmcts-reform.slack.com/messages/platops-help) channel on Slack. For example:
+
+> Hi
+> I am creating a group called "[Team name]"
```
Comment (timja) [inline source/onboarding/team/azuread.html.md.erb:22]:
```suggestion
> I am part of a new team "[Team name]"
```

File: source/ways-of-working/common-pipeline/common-pipeline.html.md.erb [hunk 4]
```diff
@@ -66,7 +66,7 @@ The environments shown in the diagram reflect CFT. Review the table below to see
 |-|-|
 |AAT|Staging|
 |Perftest|Test|
-|Preview|Dev|
+|Preview|Either Dev or Demo|
```
Comment (timja) [inline source/ways-of-working/common-pipeline/common-pipeline.html.md.erb:69]: dev reflects it, why the change here?
```suggestion
|Preview|Dev|
```

File: source/ways-of-working/common-pipeline/common-pipeline.html.md.erb [hunk 5]
```diff
@@ -89,6 +89,8 @@ End to end process:
     - updating the github flux repo with the new image name
 13. Flux will see new changes and deploy new pod
 
+In short, Jenkins runs "terraform plan" when you push, and "terraform apply" when you merge.
```
Comment (timja) [inline source/ways-of-working/common-pipeline/common-pipeline.html.md.erb:92]: not sure on this one, most runs are on the application pipeline which does a lot more than terraform plan and apply

File: source/ways-of-working/new-component/index.html.md.erb [hunk 6]
```diff
@@ -7,15 +7,19 @@ weight: 3
...
-- logging and monitoring
-- database schema management
-- [Helm chart](helm-chart.html)
-- Environment-specific concepts
-- [Flux config](gitops-flux.html#application-config-in-flux)
-- [Production approvals](https://github.com/hmcts/cnp-jenkins-config/blob/master/environment-approvals.yml)
-- [Path-to-live](../path-to-live/)
+When you create a new component (such as a database, application or a combination of such resources), you will need to follow some or all of the following:
+
+- Undertand our [infrastructure as code](infrastructure-as-code.html) approach
```
Comment (timja) [inline source/ways-of-working/new-component/index.html.md.erb:12]:
```suggestion
- Understand our [infrastructure as code](infrastructure-as-code.html) approach
```

File: source/ways-of-working/common-pipeline/common-pipeline.html.md.erb [hunk 7]
```diff
@@ -105,6 +107,30 @@ Within AKS, your application pods will exist within a namespace. The namespace w
...
+If you need to rename or destroy a resource, you will need to temporarily remove the lock. This can be done using the link in the readme of the [Azure Resource Locks](https://github.com/hmcts/azure-resource-locks/#pipeline-jobs) repository.
+
+### SSH access to resources
+
+To SSH into Azure resources, such as a Postgres Database, you will typically need to go via a Bastion.
+
+- Staging and production resources use the Production Bastion
+- Other resources use the Non-production Bastion
+
+To SSH into the bastion, you must first get a temporary access pass [here](https://myaccess.microsoft.com/@CJSCommonPlatform.onmicrosoft.com#/access-packages)
```
Comment (timja) [inline source/ways-of-working/common-pipeline/common-pipeline.html.md.erb:123]:
```suggestion
To SSH into the bastion, you must first get a temporary access pass on [myaccess.microsoft.com](https://myaccess.microsoft.com/@CJSCommonPlatform.onmicrosoft.com#/access-packages).
```

File: source/ways-of-working/common-pipeline/common-pipeline.html.md.erb [hunk 8]
```diff
@@ -105,6 +107,30 @@ Within AKS, your application pods will exist within a namespace. The namespace w
...
+
+### Using the demo environment
```
Comment (timja) [inline source/ways-of-working/common-pipeline/common-pipeline.html.md.erb:125]:
```suggestion
### Building infrastructure in the demo environment
```

File: source/ways-of-working/common-pipeline/common-pipeline.html.md.erb [hunk 9]
```diff
@@ -105,6 +107,30 @@ Within AKS, your application pods will exist within a namespace. The namespace w
...
+
+To deploy to the demo, you will simply create a branch called "demo". All merges to this branch will be deployed in the Demo environment.
```
Comment (timja) [inline source/ways-of-working/common-pipeline/common-pipeline.html.md.erb:127]:
```suggestion
To build infrastructure in demo, you will need to create a branch called "demo". All merges to this branch will be deployed in the Demo environment.
```

**Review by timja — COMMENTED** [review]
Thanks!

File: source/ways-of-working/common-pipeline/common-pipeline.html.md.erb (hunk 4 above)
Comment (WillTaylor22) [reply source/ways-of-working/common-pipeline/common-pipeline.html.md.erb:69]: Yeah great point, putting the "Either" here isn't ideal.

Well, what I thought was the dev environment I set up was actually called demo. Jenkins injected "demo" into the {env} variable, and the branch you need to name to do releases is called "demo"

I'd actually like to change this to "Demo" only, but I worried about backwards compatability if other people had something called Dev.

If we want to keep it as Dev I'm not fussed, but my experience was just a bit surprising.

**Comment by WillTaylor22** [comment]
Have merged all the changes. Only one that I would comment on is the fair objection for where I had put

"The Preview of Dev is either Dev or Demo"...

Yeah great point, putting the "Either" here isn't ideal.

Well, what I thought was the dev environment I set up was actually called demo. Jenkins injected "demo" into the {env} variable, and the branch you need to name to do releases is called "demo"

I'd actually like to change this to "Demo" only. But I worried about backwards compatibility if other people had something called Dev. It's not great reasoning.

I've changed it back to say "Dev" for now - but just something to think about.

File: source/onboarding/team/azuread.html.md.erb [hunk 10]
```diff
@@ -1,21 +1,24 @@
...
+## Creating a new team
 
-> Can you please create a team AAD group: "DTS Team name"
+If you are unsure which groups you need to be a part of, ask your team or create a help request in the [#platops-help](https://hmcts-reform.slack.com/messages/platops-help) channel on Slack. For example:
+
+> Hi
+> I am part of a new team "[Team name]"
 
-To join the groups send a pull request to the [devops-azure-aad](https://github.com/hmcts/devops-azure-ad/blob/master/users/prod_users.yml) git repository.
+> Can you please create an team AAD group: "DTS [Team name]"
```
Comment (timja) [inline source/onboarding/team/azuread.html.md.erb:24]:
```suggestion
> Can you please create a team AAD group: "DTS [Team name]"
```

**Review by timja — APPROVED** [review]
